ui/index.html

```diff
@@ -31,6 +31,9 @@
 
     <body>
         <voxetta-main></voxetta-main>
+        <button id="recordButton">Record Voice</button>
+        <audio id="utterance" controls src="" style="display: none"></audio>
+
```

Comment:
can you move all this audio stuff to its own LitElement component?

Reply:
fyi this index.html's just supposed to be a template, everything we add would be channeled thru the `vox-app` (named `voxetta-main` here)component

Reply:
In the merge conflict fix I changed it from  voxetta-main to vox-app